integration-testing/README.md

```diff
@@ -0,0 +1,32 @@
+# Integration Testing
+
+Home for integration testing different apps that need to be tested in conjuction with one another. Copy an existing subfolder if you need to set up a new bundle of apps to test.
+
+## App Bundles
+
+bsd - Tests the frontend of bsd with the backend of module-scan
+
+## Development 
```

Comment:
Should we add a note about installing Google Chrome? I didn't have it and it complained when running `pnpm test:ci`. Running `pnpm test:watch` worked, though.